docs/content/guides/upgrading.md

```diff
@@ -1,73 +1,123 @@
-# Upgrading Tenant resource from v1alpha1 to v1beta1 version
+# Capsule upgrading guide
 
-With [Capsule v0.1.0](https://github.com/clastix/capsule/releases/tag/v0.1.0), the Tenant custom resource has been bumped to `v1beta1` from `v1alpha1` with additional fields addressing the new features implemented so far.
+List of Tenant API changes:
+
+- [Capsule v0.1.0](https://github.com/clastix/capsule/releases/tag/v0.1.0) bump to `v1beta1` to `v1alpha1`.
```

Comment:
bump to `v1beta1` from `v1alpha1`
bump to `v1beta2` from `v1beta1`, deprecate `v1alpha1`

Reply:
v0.1.0 didn't deprecate `v1alpha1`.

Reply:
Fixed, thanks for the review.